packages/core-mobile/app/new/common/components/BlurViewWithFallback.tsx

```diff
@@ -33,7 +34,10 @@ export const BlurViewWithFallback = ({
   )
 
   return Platform.OS === 'ios' ? (
-    <BlurView style={iosContainerStyle} intensity={75}>
+    <BlurView
+      style={iosContainerStyle}
+      intensity={75}
+      tint={colorScheme === 'dark' ? 'dark' : 'light'}>
```

Comment:
if you make selectSelectedColorScheme only return either dark or light, we can remove this if check